docs/content/components/action-list.md

```diff
@@ -106,72 +118,76 @@ List item `li` for separating groups of content
 
 ```html live
 <ul class="ActionList">
-  <li class="ActionList-sectionDivider" role="separator"></li>
+  <span
+    class="ActionList-sectionDivider"
+    role="separator"
+    aria-hidden="true"
+  ></span>
```

Comment:
I think it's invalid HTML to put the `<span>` directly under `<ul>` [ref](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul)

Reply:
Fixed!